### PR TITLE
Richlink byline pics

### PR DESCRIPTION
**Pull request by sndrs**
# My Awesome Pull Request
## What does this change?

rich links for comment pieces which are also special reports were not really usuable, because they weren't picking up the comment features. this changes richlinks to use `hasTonalHeaderByline` instead of `content.content.cardStyle == Comment`, and adds some styling to special report tone
## Request for comment

@rich-nguyen @hoyla
## Screenshots

richlinks mobile
richlinks tablet
richlinks desktop
richlinks wide
## 

_Does this PR meet the [contributing guidelines](https://github.com/guardian/frontend/blob/issue_pr_templates/.github/CONTRIBUTING.md#submission)?_

### CODE REVIEW

File: onward/app/views/fragments/richLinkBody.scala.html [hunk 1]
```diff
@@ -38,12 +38,12 @@
 
         <div class="rich-link__header">
             <h1 class="rich-link__title">
-                @if( toneClass(content) == "tone-comment" ) {
+                @if( content.content.hasTonalHeaderByline ) {
```
Comment (sndrs) [inline onward/app/views/fragments/richLinkBody.scala.html]: this is adding it to editorial tones too, which it shoudln't,  will update

**Comment by sndrs** [comment]
fixed editorial pieces picking up the quote treatments. it's now limited to `comment` or `special-report+hasTonalHeaderByline`

screen shot 2016-04-06 at 14 35 37
screen shot 2016-04-06 at 14 35 24
screen shot 2016-04-06 at 14 35 04

**Comment by sndrs** [comment]
fix comment pieces getting forced height if there's no byline pic

### before
screen shot 2016-04-06 at 14 35 37

### after
screen shot 2016-04-06 at 14 45 01

**Comment by rich-nguyen** [comment]
:+1:

**Pull request merged**